### PR TITLE
Ecs prod

### PR DESCRIPTION
**Pull request by aaronFranssell**
### JIRA Ticket

SDC-1939

### Summary

Add in ECS prod configs.

### CODE REVIEW

File: webportal/lambda/.chalice/config.json [hunk 1]
```diff
@@ -94,18 +94,18 @@
...
+        "RESTAPIID": "3jca2s255l",
+        "AUTHORIZERID": "tvt9se",
+        "TABLENAME_TRUSTED_USERS": "production-TrustedUsersTable",
+        "TABLENAME_EXPORT_FILE_REQUEST": "production-RequestExportTable",
+        "TABLENAME_MANAGE_USER": "production-ManageUserWorkstationTable",
         "TABLENAME_MANAGE_USER_INDEX": "prod-workstation-username-index",
         "TABLENAME_MANAGE_DISK": "prod-ManageDiskspaceRequestsTable",
         "TABLENAME_MANAGE_DISK_INDEX": "prod-diskspace-username-index",
-        "TABLENAME_MANAGE_UPTIME": "prod-ScheduleUptimeTable",
+        "TABLENAME_MANAGE_UPTIME": "production-ScheduleUptimeTable",
```
Comment (JAMSUPREME) [inline webportal/lambda/.chalice/config.json:105]: We bouncing between `prod` and `production` again?

**Review by JAMSUPREME — DISMISSED** [review]
👍

File: webportal/lambda/.chalice/config.json (hunk 1 above)
Comment (aaronFranssell) [reply webportal/lambda/.chalice/config.json:105]: Yes :-(

**Review by JAMSUPREME — APPROVED** [review]
👍

**Pull request merged**